pkg/cloudfoundry/ManifestUtils_test.go

```diff
@@ -0,0 +1,111 @@
+package cloudfoundry
+
+import (
+	"testing"
+
+	"fmt"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestReadManifest(t *testing.T) {
+
+	_readFile = func(filename string) ([]byte, error) {
+		if filename == "myManifest.yaml" {
+			return []byte("applications: [{name: 'manifestAppName'}]"), nil
+		}
+		return []byte{}, fmt.Errorf("File '%s' not found", filename)
+	}
+
```

Comment:
You are not resetting _readFile pointer. This could result in flaky tests. Also, as already commented, modifying function pointers prevents parallel test execution.

Reply:
I took care that each method comes with its own function. But for later cases (... extending the tests) it is beneficial to clean up. I will provide a cleanup defer func.

Reply:
... done, we have a defered cleanup method now